client/src/components/CalendarHeader/CalendarHeader.module.css

```diff
@@ -0,0 +1,27 @@
+.calendarPageHeader {
+  color: white;
+  font-weight: 600;
+  font-family: "Source Sans Pro", sans-serif;
+  font-size: 1.875rem;
+  margin: 5px 10px;
+}
+
+.calendarHeaderContainer {
+  display: flex;
+  background: #032a5c;
+  align-items: center;
+}
+
+.hamburgerIcon {
+  height: 1.875rem;
+  width: 1.875rem;
+  background: transparent;
+  color: #dadada;
+}
+
+.navButton {
+  background: transparent;
+  border: none;
+  margin: 0px 0px 0px 10px;
+  padding: 0.4rem 0px 0px 0px;
```

Comment:
Same comment as above (with the margin)
`padding: 0.4rem 0px 0px 0px`
is the same as 
`padding-top: 0.4rem`

Reply:
Additionally, I wouldn't use `rem` for padding

Reply:
I would suggest using `em` as I believe it's more applicable instead of `rem`
https://www.geeksforgeeks.org/difference-between-em-and-rem-units-in-css/
https://dev.to/datmt/use-em-for-padding-and-margin-54bi

Reply:
> Same comment as above (with the margin) `padding: 0.4rem 0px 0px 0px` is the same as `padding-top: 0.4rem`

Fixed code style

Reply:
> I would suggest using `em` as I believe it's more applicable instead of `rem` (At least for now, we can stick with `px` as well) https://www.geeksforgeeks.org/difference-between-em-and-rem-units-in-css/ https://dev.to/datmt/use-em-for-padding-and-margin-54bi

Changed to use `em`.